ci/common/suite.sh

```diff
@@ -45,23 +45,23 @@ fail() {
 
 run_test() {
   local cmd="$1"
-  shift
+  test $# -gt 0 && shift
   local test_name="$1"
   : ${test_name:=$cmd}
-  shift
+  test $# -gt 0 && shift
   if ! eval "$cmd" ; then
     fail "${test_name}" "$@"
   fi
 }
 
 run_test_wd() {
   local timeout="$1"
-  shift
+  test $# -gt 0 && shift
   local cmd="$1"
-  shift
+  test $# -gt 0 && shift
   local test_name="$1"
```

Comment:
And repetive `test $# -gt 0 && shift` will shift up until no arguments left. So test_name will have the same value only if first argument was missing, and further code handles empty `$1`.